index.bs

```diff
@@ -638,3 +646,134 @@ registerAnimator('spring', class {
     }
 });
 </pre>
+
+
+Example 2: Twitter header. {#example-2}
+--------------------------
+
+
+<pre class='lang-markup'>
+&lt;style&gt;
+#scroller {
+  --animator-root: twitter-header;
```

Comment:
For the polyfill version I used --animator-root and --animator because custom properties can fill in for those missing css properties, but for the spec examples this should not have the leading --.